kornia/geometry/depth.py

```diff
@@ -365,3 +366,36 @@ def depth_warp(
     warper = DepthWarper(pinhole_dst, height, width, align_corners=align_corners)
     warper.compute_projection_matrix(pinhole_src)
     return warper(depth_src, patch_dst)
+
+
+def depth_from_disparity(disparity: Tensor, baseline: Union[float, Tensor], focal: Union[float, Tensor]) -> Tensor:
+    """Computes depth from disparity.
+
+    Args:
+        disparity: Disparity tensor of shape :math:`(B, 1, H, W)`.
+        baseline: float/tensor containing the distance between the two lenses.
+        focal: float/tensor containing the focal length.
+
+    Return:
+        Depth map of the shape :math:`(B, 1, H, W)`.
+
+    Example:
+        >>> disparity = torch.rand(1, 1, 4, 4)
+        >>> baseline = torch.rand(1)
+        >>> focal = torch.rand(1)
+        >>> depth_from_disparity(disparity, baseline, focal).shape
+        torch.Size([1, 1, 4, 4])
+    """
+    KORNIA_CHECK_IS_TENSOR(disparity, f"Input disparity type is not a Tensor. Got {type(disparity)}.")
+    KORNIA_CHECK_SHAPE(disparity, ["B", "1", "H", "W"])
+    KORNIA_CHECK(
+        isinstance(baseline, (float, Tensor)),
+        f"Input baseline should be either a float or Tensor. " f"Got {type(baseline)}",
+    )
+    KORNIA_CHECK_SHAPE(baseline, ["1"])
```

Comment:
Now we have still some limitations by torchscript that prevents us from checking shapes other than Tensors

Reply:
Ah my mistake, I intended to add it along with isTensor test but it slipped my mind. WIll fix it!

Reply:
But it could be equal to the batch_size too, so how do I check for that?

Reply:
```python
if isinstance(base_line, Tensor):
   KORNIA_CHECK(disparity.shape[:-2] == baseline.shape)
```

or something like that